src/views/CrosswordpuzzleView.vue

```diff
@@ -102,6 +102,7 @@ function getWrongQuestionVertical(element: TileCrossWord): questionAnswer {
 }
 
 function evaluateSolution() {
+  //NOSONAR
```

Comment:
why exactly?

Reply:
Because this function is too large?
The better alternative would be to extract subfunctions.